pyqtgraph/graphicsItems/ScatterPlotItem.py

```diff
@@ -224,13 +227,16 @@ def diagnostics(self):
                     squareness=1.0 if n == 0 else 2 * w * h / (w**2 + h**2))
 
     def _keys(self, styles):
+        def getId(obj):
+            try:
+                return obj._id
+            except AttributeError:
+                obj._id = next(self._idGenerator)
```

Comment:
should this be `next(SymbolAtlas._idGenerator)` as it is a class variable and not an instance variable?

Reply:
Yes, thanks